languages/translations/sv.json

```diff
@@ -84,28 +84,41 @@
         },
         "continue": {
             "default": {
-                "default": "Fortsätt {modifier}",
-                "name": "Fortsätt {modifier} in på {way_name}",
-                "destination": "Fortsätt {modifier} mot {destination}"
+                "default": "Sväng {modifier}",
+                "name": "Turn {modifier} to stay on {way_name}",
+                "destination": "Sväng {modifier} mot {destination}",
+                "exit": "Sväng {modifier} in på {way_name}"
             },
             "straight": {
                 "default": "Fortsätt rakt fram",
-                "name": "Fortsätt in på {way_name}",
-                "destination": "Fortsätt mot {destination}"
+                "name": "Continue straight to stay on {way_name}",
+                "destination": "Fortsätt mot {destination}",
+                "distance": "Continue straight for {distance}",
+                "namedistance": "Continue on {way_name} for {distance}"
+            },
+            "sharp left": {
+                "default": "Gör en skarp vänstersväng",
+                "name": "Make a sharp left to stay on {way_name}",
+                "destination": "Gör en skarp vänstersväng mot {destination}"
+            },
+            "sharp right": {
+                "default": "Gör en skarp högersväng",
+                "name": "Make a sharp right to stay on {way_name}",
+                "destination": "Gör en skarp högersväng mot {destination}"
             },
             "slight left": {
-                "default": "Fortsätt åt vänster",
-                "name": "Fortsätt åt vänster in på {way_name}",
-                "destination": "Fortsätt åt vänster mot {destination}"
+                "default": "Make a slight left",
```

Comment:
Same here, english. several of the continue has a history of having a translation previously. But now they are empty and untranslated.